tiledb/cloud/config.py

```diff
@@ -12,6 +12,13 @@
 default_config_file = Path.joinpath(Path.home(), ".tiledb", "cloud.json")
 
 
+def parse_bool(s: str) -> bool:
+    if s is None:
```

Comment:
This should never happen per the type annotation. If you expect to get `None` values, then `s` should be an `Optional[str]`.